src/Sulu/Bundle/SnippetBundle/Document/SnippetDocument.php

```diff
@@ -51,10 +53,15 @@ class SnippetDocument implements
     private $locale;
     private $path;
     private $nodeName;
+    private $synchronizedManagers;
 
     public function __construct()
     {
-        $this->workflowStage = WorkflowStage::TEST;
+        // snippets should be published by default as there is no way
+        // to change this in the UI and this is the effective state.
+        //
+        // TODO: write a migration for this?
+        $this->workflowStage = WorkflowStage::PUBLISHED;
```

Comment:
Write  a migration for this? I don't think this comment is even valid anymore (as it would be up to the snippet controller to "publish" the snippet or not, or maybe there should be a way to automatically publish?).

In anycase, changing this value to PUBLISHED is more correct as snippets are published by default atm.